### ISSUE TITLE
Database: баги, сервер не запкскается

### ISSUE THREAD

**Issue by lounres** [m20-sch57/labyrinth#206]
Решил я грохнуть БД, чтобы удалить мусорные данные, начать тестирование с чистого листа и т.п. Но при запуске сервака и открывании `localhost:3000` и `localhost:3000/index` получается вот это:
![изображение](https://user-images.githubusercontent.com/43728100/59129895-ad690d00-8976-11e9-8961-4da3f22043c5.png)

**Comment by gtivansan** [comment]
У тебя в сессии сохранилось имя пользователя. Просто перейди на` /logout`

**Comment by lounres** [comment]
Да, спасибо, работает. Но мне кажется, что это стоит всё-таки как-то исправить с нашей стороны, а не с пользовательской (в случаях, когда такое будет появляться в будущем).

**Comment by gtivansan** [comment]
В теории, такое не должно появляться. Нельзя же пользователей удалять из бд.
Но можно и исправить.

**Comment by lounres** [comment]
Ну, я и рассчитываю, что это можно исправить.

P.S. Я не понимаю, что произошло.

**Comment by gtivansan** [comment]
Пока ты не вышел, в твоей сессии лежит username, которого нет в бд. А мы считаем, что username в сессии - это гарантия того, что он там есть и что пользователь вошёл.

**Comment by lounres** [comment]
А где хранится сессия? У сервера?

**Comment by lounres** [comment]
А тут ответа нет...

**Comment by gtivansan** [comment]
Сори, забыл.

Да сессия хранится на сервере. А что?

**Comment by gtivansan** [comment]
Если что, чтобы исправить эту ошибку нужно везде добавить костыль:

если username из сессии не в бд, то сбросить сессию.

**Comment by lounres** [comment]
Понял. Просто интересно. Я пишу только движок, поэтому совершенно не знаю, как что-либо другое тут работает.
